webui/react/src/services/api.ts

```diff
@@ -109,35 +109,17 @@ export const updateJobQueue = generateDetApi<
 >(Config.updateJobQueue);
 
 /* Trials */
-export const queryTrials = generateDetApi<
-  Api.V1QueryTrialsRequest,
-  Api.V1QueryTrialsResponse,
-  Api.V1QueryTrialsResponse
->(Config.queryTrials);
-
-export const patchTrials = generateDetApi<
-  Api.V1PatchTrialsRequest,
-  Api.V1PatchTrialsResponse,
-  Api.V1PatchTrialsResponse
->(Config.patchTrials);
-
-export const createTrialsCollection = generateDetApi<
-  Api.V1CreateTrialsCollectionRequest,
-  Api.V1CreateTrialsCollectionResponse,
-  TrialsCollection | undefined
->(Config.createTrialCollection);
-
-export const getTrialsCollections = generateDetApi<
-  number,
-  Api.V1GetTrialsCollectionsResponse,
-  Api.V1GetTrialsCollectionsResponse
->(Config.getTrialsCollections);
-
-export const patchTrialsCollection = generateDetApi<
-Api.V1PatchTrialsCollectionRequest,
-Api.V1PatchTrialsCollectionResponse,
-TrialsCollection | undefined
->(Config.patchTrialCollection);
```

Comment:
apparently all these type arguments throughout the file are redundant since the types get inferred. 

would be nice to get rid of all of them :)